final_task/pycalc/__main__.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/python3
+# -*- coding: UTF-8 -*-
+
+# general import
+import argparse
+import sys
+# import from pycalc self library
+from .tokenizer import Tokenizer
+from .addmultsigns import Multsignsadder
+from .rpn import RPN
+from .constsreplacer import Constsreplacer
+from .rpncalculator import RPNcalculator
+
+
+def createparser():
```

Comment:
I suggest to rename it like `create_parser`. See naming conventions here: https://www.python.org/dev/peps/pep-0008/#naming-conventions

Reply:
fixed